src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer_GetDiagnostics.cs

```diff
@@ -399,7 +399,8 @@ protected override bool ShouldIncludeDiagnostic(DiagnosticData diagnostic)
                 // unlike the suppressed (disabled) analyzer, we will include hidden diagnostic only analyzers here.
                 var analyzerDriverOpt = await Owner._compilationManager.CreateAnalyzerDriverAsync(project, stateSets, IncludeSuppressedDiagnostics, cancellationToken).ConfigureAwait(false);
 
-                var result = await Owner._executor.GetProjectAnalysisDataAsync(analyzerDriverOpt, project, stateSets, cancellationToken).ConfigureAwait(false);
+                var ignoreOption = true;
```

Comment:
Can you add a comment here and probably rename the flag to ignoreClosedFileDiagnosticsOption?